src/helix-ui/elements/HXFileTileElement.less

```diff
@@ -163,12 +163,12 @@
 
 :host([progress]) {
   #hxLink {
-    color: @gray-600;
+    color: var(--hxColor, @gray-600);
   }
 
   #hxIconWrapper,
   #hxName {
-    color: @gray-750;
+    color: var(--hxColor, @gray-750);
```

Comment:
* might need to split the selector into two blocks with separate custom properties

```less
#hxIconWrapper {
  color: var(--hxIcon-color, @gray-750);
}
#hxName {
  color: var(--hxName-color, @gray-750);
}
```